receiver/k8sclusterreceiver/watcher.go

```diff
@@ -153,6 +176,16 @@ func (rw *resourceWatcher) prepareSharedInformerFactory() error {
 	}
 	rw.informerFactories = append(rw.informerFactories, factory)
 
+	isSupported, _ := rw.isKindSupported(gvk.HierarchicalResourceQuota)
```

Comment:
Let's handle this error like we do other `isKindSupported` errors in this function.  If the user configures these metrics but the receiver can't get them then thats an issue.

Reply:
I fixed to handle errors.